hallux/backends/factory.py

```diff
@@ -36,6 +44,40 @@ def init_backend(argv: list[str], config: list[dict] | None, config_path: Path)
                 return backend  # stop early if required by CLI
         return backend
 
+    @staticmethod
+    def validate_model_args(argv, model_index: int) -> str | None:
+        model_value = find_argvalue(argv, "--model")
+        if model_value and model_value != "":
+            # Ensure the last argument is a valid path
+
+            has_delimeter = "=" in argv[model_index]
+            max_args = model_index + (1 if has_delimeter else 2)
+            is_extra_arg_present = len(argv) > max_args
+            is_last_arg_not_option = not argv[-1].startswith("--")
+
+            if is_extra_arg_present and is_last_arg_not_option:
+                path_value = argv[-1]
+                if path_value:
+                    return model_value
+                else:
+                    raise SystemError("Missing last path argument. Please provide a valid path")
+                    # logger.error("Missing last path argument. Please provide a valid path.")
+                    # return None
+            else:
+                raise SystemError("Missing or invalid last path argument. Please provide a valid path.")
+                # logger.error("Missing or invalid last path argument. Please provide a valid path.")
+                # return None
```

Comment:
Remove this commented out code.
```suggestion
            else:
                raise SystemError("Missing or invalid last path argument. Please provide a valid path.")
```